### DISCUSSION TITLE
Text Field Component Size

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#6864] [Help] [answered]
Hello, 

Is there any way to re-size the "New Text Field" to a bigger size?
I really want something that resembles notepad so the user doesn't have to navigate thru the text.

![image](https://user-images.githubusercontent.com/107686004/180472244-c970104f-a9bc-47d0-b30d-6a14bcd1f725.png)

Thank you in advance!

**Comment by mmalessa80** [comment]
Apparently adding the proper CSS adds the styling for a draggable corner, however it does not seem to work correctly. Tried both on a text input and a long form field. Might be more dependent on a <textarea> field vs input.

![image](https://user-images.githubusercontent.com/106758851/180495666-271bb9fe-a694-41e1-9fb4-1cc5394ef756.png)

**Comment by melohagan** [comment, marked as answer]
Hello @insomniacj93 

You probably want to use a **Long Form Field**. It can be resized by dragging the bottom right hand corner.

https://docs.budibase.com/docs/long-form-field#resizing
